seatunnel-ui/.env.development

```diff
@@ -15,4 +15,4 @@
 
 NODE_ENV=development
 
-VITE_APP_DEV_WEB_URL=''
+VITE_APP_DEV_WEB_URL='http://127.0.0.1:8080'
```

Comment:
How about change the port? `8080` is  too common and easy to conflict.

Reply:
done.